sources/platform/integrations/flowise.md

```diff
@@ -9,28 +9,50 @@ slug: /integrations/flowise
 
 ---
 
-> For more information on Flowise, visit [documentation](https://flowiseai.com/).
-
 ## What is Flowise?
 
 Flowise is an open-source UI visual tool to build your customized LLM flow using Langchain.
 
-## How to use Apify with Flowise?
+## How to use Apify with Flowise
+
+### Installation
+
+To use Flowise you have to download and run it locally. The quickest way to do so is to use the following commands:
+
+1. To install Flowise globally on your device:
+
+    ```bash
+    npm install -g flowise
+    ```
+
+2. To start Flowise locally:
+
+    ```bash
+    npx flowise start
+    ```
+
+It will be available on `https://localhost:3000`
 
-If you have any experience with Flowise, you need to download and run it locally.
-You can find the instructions [here](https://github.com/FlowiseAI/Flowise#quick-start).
+Other methods of using Flowise can be found in their [documentation](https://docs.flowiseai.com/getting-started#quick-start)
+
+### Building your flow
```

Comment:
```suggestion
### Building your Flow
```

This is kinda off topic and I'm not even sure what I suggest is the correct capitalization. Do we conform to some particular style here?

Reply:
I think if Flowise ever referred to it with capitalization we would do it too, but I checked out their docs and haven't seen them referring to it as Flow so I would go without capitalization here